src/metrics.jl

```diff
@@ -55,8 +56,15 @@ type JSDivergence <: SemiMetric end
 
 type SpanNormDist <: SemiMetric end
 
+# Deviations are handled separately from the other distances/divergences and
+# are excluded from `UnionMetrics`
+type MeanAbsDeviation <: Metric end
+type MeanSqDeviation <: SemiMetric end
+type RMSDeviation <: Metric end
+type NormRMSDeviation <: Metric end
```

Comment:
Now I look at this, NormRMSDeviation (and CVRMSDeviation, below) are premetrics as they are not symmetric (since they are normalised by only one of their inputs).